build/build-image/Dockerfile

```diff
@@ -25,7 +25,8 @@ RUN chmod -R a+rwx /usr/local/go/pkg
 # of operations.
 ENV HOME /go/src/k8s.io/kubernetes
 WORKDIR ${HOME}
-RUN chmod -R a+rwx ${HOME}
+RUN mkdir -p ${HOME} ${HOME}/_output \
```

Comment:
Sadly, docker is just sort of brain damaged.  If I don't mkdir _output, docker will mkdir it for me, but it will do so as root.  I need to be able to write to it.  If I don't chmod ir explicitly, it does not stick.  I can't explain why, and I didn't want to waste more time on it.  Will comment.